charts/kong-operator/values.yaml

```diff
@@ -0,0 +1,114 @@
+image:
+  repository: docker.io/kong/gateway-operator
```

Comment:
Would we update this to `docker.io/kong/gateway/operator` when the docker repo is ready? I think there should be a comment to tell us that we are using `docker.io/kong/gateway-operator` image here before the `kong-operator` image is ready for CI.

Reply:
Agreed. 54550b8e28609018bc8dab0644a37094b5d6d011 covers that (and links to #1598).